product_docs/docs/biganimal/release/getting_started/preparing_cloud_account/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -41,30 +37,26 @@ The default public IP addresses limits for basic and standard type are set to 10
 
 ### Recommended limit
 
-EDB recommends that you increase the public IP addresses limits for Basic and Standard type to 50. If you need more than 50 clusters, increase the limit to the number of clusters you plan to deploy.
+This is decided by the number of the clusters you will create. 
 
 ## vCPU limits
 
 Any time a new VM is deployed in Azure, the vCPUs for the VMs must not exceed the total vCPU limits for the region.
 
-Clusters deployed in the region use ESv3 virtual machine cores. The number of cores depends on the instance type and HA options of the clusters. You can calculate the number of ESv3 cores required for your cluster based on the following:
+The number of cores required by the database cluster depends on the instance type and cluster type of the clusters. For exampe, if you create cluster with ESv3 instance type, you can calculate the number of ESv3 cores required for your cluster based on the following:
 
 -   A virtual machine instance of type E{N}sv3 uses {N} cores. For example, an instance of type E64sv3 uses 64 ESv3 cores.
 -   A cluster running on an E{N}sv3 instance without HA enabled uses exactly {N} ESv3 cores.
--   A cluster running on an E{N}sv3 instance with HA enabled uses 3 \* {N} ESv3 cores.
-
-For example, if you provision the largest virtual machine E64sv3 with high availability enabled, it requires (3 \* 64) = 192 ESv3 cores per region.
-
-BigAnimal requires an additional eight Dv4 virtual machine cores per region.
+-   A cluster running on an E{N}sv3 instance with HA enabled and 2 replicas uses 3 \* {N} ESv3 cores.
 
-### Default limit
+BigAnimal requires an additional eight Dv4 virtual machine cores per region for management resources. 
 
-The default number of total vCPU (cores) per subscription per region is 20. For more information, see [Virtual Machines limits - Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#virtual-machines-limits---azure-resource-manager).
+As the AKS cluster involves performing periodic upgrades to the proper Kubernetes version, an additional six Dv4 virtual machine cores per region are needed for AKS upgrade.
```

Comment:
We haven't really introduced the concept of the AKS cluster in the docs. The only place we mention AKS is under pricing and in metrics labels so it might cause some confusion. Can we say instead:

> BigAnimal requires an additional six Dv4 virtual machine cores per region for periodic maintenance upgrades.

Or, would it be possible to include the number in the previous paragraph and say:

> BigAnimal requires an additional 14 Dv4 virtual machine cores per region for management resources. 

AKS is listed under management costs on the Pricing page.

Reply:
I prefer the first one as these additional six Dv4 are not always required, they are only required during upgrade.